content/de/blog/hacktoberfest-2021/index.md

```diff
@@ -0,0 +1,121 @@
+{
+    "title": "Hacktoberfest 2021: Macht mit!",
+    "type": "blog",
+    "date": "2021-09-19T11:00:0+02:00",
+    "description": "Wir halten 2021 auch wieder eine Hacktoberfest-Aktion ab. Mach' bei Datenanfragen.de mit und gewinne einen Webcam-Slider, Sticker und vielleicht ein T-Shirt!",
```

Comment:
Not too sure about the term "Webcam-Slider", I don't think that's commonly understood. I'd have used "Webcam-Abdeckung".